libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryLocationAndProgressDispatcherImpl.kt

```diff
@@ -0,0 +1,150 @@
+package com.mapbox.navigation.core.telemetry
+
+import android.location.Location
+import android.util.Log
```

Comment:
For now, I'd prefer using the `Logger` (instead of removing the logs) as we're still tracking some tailwork and logs help a lot debugging, testing and confirming that the implementation fits and covers all the uses cases / scenarios.

Reply:
done